client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -476,19 +477,35 @@ export class PlanFeatures2023Grid extends Component<
 					) }
 					<header className={ headerClasses }>
 						{ isBusinessPlan( planName ) && (
-							<div className="plan-features-2023-grid__plan-logo">
-								<img src={ cloudLogo } alt="Cloud logo" />{ ' ' }
-							</div>
+							<Plans2023Tooltip
+								text={ translate(
+									'WP Cloud gives you the tools you need to add scalable, highly available, extremely fast WordPress hosting.'
```

Comment:
I think this name should change to Business instead of WP Cloud. Hovering over the logo is somewhat confusing to see `WP Cloud` when it is not mentioned anywhere else.

WDYT @lucasmendes-design @michaelpick @southp

Reply:
I still have some concern regarding it and I've raised it here: p2-p7DVsv-gse#comment-44359. If it's fine for you, let's wait for the answer before doing so :)

Reply:
The logo means that the plan is powered by WP Cloud. I believe this is the intention when Matias asked for that. I would keep it.